lib/services/authentication.dart

```diff
@@ -1,11 +1,12 @@
 import 'dart:async';
 import 'package:firebase_auth/firebase_auth.dart';
-import 'package:hoopr/services/user.dart';
+import 'package:hoopr/services/db.dart';
```

Comment:
Update this to `package:hoopr/services/database.dart` if we change that file name